exe/api_log_parser

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env bundle exec ruby
```

Comment:
I've never seen a shebang declared in this way...did you try using the more typical `#!/usr/bin/env ruby` and following the steps outlined in https://bundler.io/v1.16/guides/bundler_setup.html ? That way you may also be able to tag only the gems you need so that only they are required when you setup bundle - which should improve the load time significantly.